jp2_pc/Source/Lib/EntityDBase/MessageTypes.cpp

```diff
@@ -769,5 +769,5 @@ namespace
 	{
 		CMessageAudio* pmsg_new = new(frhFrameHeap) CMessageAudio();
 		*pmsg_new = *this;
-		qmQueueMessage.pdqwmNextMessages->push(pmsg_new);
+		qmQueueMessage.pdqwmNextMessages->push({ pmsg_new });
 	}
```

Comment:
We try to insert a `std::deque` with `push()`, which is constructed from an initializer list.